doc/source/ray-observability/user-guides/configure-logging.md

```diff
@@ -45,8 +45,9 @@ System logs may include information about your applications. For example, ``runt
 - ``worker-[worker_id]-[job_id]-[pid].[out|err]``: Python or Java part of Ray drivers and workers. Ray streams all stdout and stderr from Tasks or Actors to these files. Note that job_id is the ID of the driver.
 
 ### System/component logs
-- ``dashboard.[log|err]``: A log file of a Ray Dashboard. ``.log`` files contain logs generated from the dashboard's logger. ``.err`` files contain stdout and stderr printed from the dashboard. They're usually empty except when the dashboard crashes unexpectedly.
+- ``dashboard.[log|err]``: A log file of a Ray Dashboard. ``.log`` files contain logs generated from the dashboard's logger. ``.err`` files contain stdout and stderr printed from the dashboard. They're usually empty except when the dashboard crashes unexpectedly. For subprocess modules, the logs of the parent process also go here.
```

Comment:
> For subprocess modules, the logs of the parent process also go here.

Do you mean subprocess stderr also goes to dashboard.err?

Reply:
No. The stderr for parent process goes here. And the stderr for child processes go to `dashboard_[module_name].log`

Reply:
Updated. After https://github.com/ray-project/ray/pull/51545 the stdout and stderr for child processes will go to `dashboard_[module_name].err`.